packages/node-xmpp-client/lib/bosh.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+require('es6-collections')
```

Comment:
Fixed! 

I noticed that you have this require in node-xmpp-server. Probably you want to remove it .